From e451dcf6f866e599d61c2614524773f349f97e34 Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Mon, 2 Jun 2014 15:23:04 -0400
Subject: [redhat] Kconfig: enable Intel PowerClamp for x86_64

Message-id: <20140602152304.16229.85536.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 81564
O-Subject: [PATCH RHEL7.1 BZ1103376 4/4] redhat Kconfig: enable Intel PowerClamp for x86_64
Bugzilla: 1103376
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1103376

Description:
------------
Enable Intel PowerClamp Driver for x86_64.

RHEL Version Found:
-------------------
7.0

Upstream Status:
----------------
n/a

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_INTEL_POWERCLAMP b/redhat/configs/generic/x86_64/CONFIG_INTEL_POWERCLAMP
index c0738e6..84f4db2 100644
--- a/redhat/configs/generic/x86_64/CONFIG_INTEL_POWERCLAMP
+++ b/redhat/configs/generic/x86_64/CONFIG_INTEL_POWERCLAMP
@@ -1 +1 @@
-# CONFIG_INTEL_POWERCLAMP is not set
+CONFIG_INTEL_POWERCLAMP=m
-- 
1.7.1